capture/split.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import logging
+
+from Bio import SeqIO
+from capture import bam
+from capture import parse
+
+
+def split(args, file, type_f, wanted_cov=50):
```

Comment:
you don't want to pass around the `args` object around in your function; it makes unit testing very difficult. you should pass mean and genome_size explicitly